grace/evaluation/visualisation.py

```diff
@@ -0,0 +1,258 @@
+from grace.base import GraphAttrs, Annotation
+from grace.napari.utils import EdgeColor
+from collections import Counter
+
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+
+import numpy.typing as npt
+
+from skimage.util import montage
+
+
+def plot_simple_graph(G: nx.Graph, title: str = "") -> None:
+    """Plots a simple graph with black nodes and edges."""
```

Comment:
Add function input parameters

Reply:
Input params are typed in directly, there's not much more to describe here as their names are self-explanatory.